app/pastes/get-paste-by-hash/getPasteByHashInteractor.js

```diff
@@ -0,0 +1,42 @@
+const NotFound = require('../../errors/notFound');
+const ValidationError = require('../../errors/validationError');
+const ApiError = require('../../errors/apiError');
+
+class GetPasteByHashInteractor {
```

Comment:
I think we should rename the whole use-case into something like "GetSharedPaste", without exposing details of how we do it (today we use token-hashes, tomorrow we might implement some kind of dynamic id, etc)